play/apps/snake/templates/snakes/index.html

```diff
@@ -126,11 +131,23 @@ <h5 class="modal-title">Delete Snake</h5>
     trigger: 'click',
     title: 'Snake API Validation <button type="button" class="close" aria-label="Close" style="line-height: .6;"><span aria-hidden="true">&times;</span></button>',
     html: true,
-    content: snakeStatuses[snakeUrl],
+    content: snakeStatuses[snakeUrl] || status('Snake is currently being validated ...', 'info')
   })
   $elem.popover('show')
 })
 
+function status(message, type) {
+  return `<div class="alert alert-${type} text-center">${message}</div>`
+}
+
+function changeInfoButtonState(state) {
```

Comment:
This causes all the info buttons to fail, not just the one that is having a validation problem. The function needs to scope to just the correct snake being validated.

Reply:
@dlsteuer Thanks, I missed this as I only tested with one snake.  Updated